### PR TITLE
Webpack setup

### PR DESCRIPTION
**Pull request by shaitaldesai**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+var path = require('path');
+var APP_DIR = path.join(__dirname, 'client/src');
+var BUILD_DIR = path.join(__dirname, 'client/dist');
+
+module.exports = {
+  entry: `${APP_DIR}/index.js`,
```
Comment (lee-philip-n) [inline webpack.config.js]: Since you are using react, You should change this from index.js to index.jsx

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  output: {
+  	path: BUILD_DIR,
+    filename: 'bundle.js'
+  },
+	module: {
+	  rules: [
+	    { test: /\.js$/, exclude: /node_modules/, loader: "babel-loader" }
```
Comment (lee-philip-n) [inline webpack.config.js]: This will need to be changed as well to .jsx?$

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules
+client/dist/bundle.js
```
Comment (lee-philip-n) [inline .gitignore]: need to also gitignore .DS_Store

**Comment by shaitaldesai** [comment]
https://trello.com/c/4W2XkWN3/4-front-end-setup-webpack-react-babel-m

File: data.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+var faker = require('faker');
```
Comment (lee-philip-n) [inline data.js]: this is not part of the module, remove it

**Comment by lee-philip-n** [comment]
I made some comments on the files changed section. Reopen the PR once you have completed them.

**Comment by shaitaldesai** [comment]
Refactored .gitignore and webpack.config.js as requested.

**Comment by shaitaldesai** [comment]
Refactored .gitignore and webpack.config.js as requested.

**Comment by shaitaldesai** [comment]
Refactored .gitignore and webpack.config.js as requested.

**Comment by shaitaldesai** [comment]
Refactored .gitignore and webpack.config.js as requested.
https://trello.com/c/4W2XkWN3/4-front-end-setup-webpack-react-babel-m

**Pull request merged**